plotly_resampler/aggregation/aggregators.py

```diff
@@ -181,56 +161,46 @@ class MinMaxAggregator(AbstractSeriesAggregator):
 
     """
 
-    def __init__(self, interleave_gaps: bool = True, nan_position="end"):
+    def __init__(self, interleave_gaps: bool = True):
         """
         Parameters
         ----------
         interleave_gaps: bool, optional
             Whether None values should be added when there are gaps / irregularly
             sampled data. A quantile-based approach is used to determine the gaps /
             irregularly sampled data. By default, True.
-        nan_position: str, optional
-            Indicates where nans must be placed when gaps are detected. \n
-            If ``'end'``, the first point after a gap will be replaced with a
-            nan-value \n
-            If ``'begin'``, the last point before a gap will be replaced with a
-            nan-value \n
-            If ``'both'``, both the encompassing gap datapoints are replaced with
-            nan-values \n
-            .. note::
-                This parameter only has an effect when ``interleave_gaps`` is set
-                to *True*.
-        dtype_regex_list: List[str], optional
-            List containing the regex matching the supported datatypes, by default None.
+
         """
-        # this downsampler supports all pd.Series dtypes
-        super().__init__(interleave_gaps, nan_position, dtype_regex_list=None)
+        # this downsampler supports all dtypes
+        super().__init__(interleave_gaps)
 
-    def _aggregate(self, s: pd.Series, n_out: int) -> pd.Series:
+    def _arg_downsample(
+        self,
+        x: Optional[np.ndarray] = None,
+        y: Optional[np.ndarray] = None,
```

Comment:
Some of the other downsamples have different types, I think - let's check they're consistent.